templates/patterns/icons_with_text/icons_with_text.ui_patterns.yml

```diff
@@ -8,11 +8,13 @@ icons_with_text:
       description: "List of items, each having an 'icon' and a 'text' field"
       preview:
         - icon: "file"
+          size: "s"
```

Comment:
Might be we don't need this line. Size `m` will be used by default.

Reply:
Let's leave it so we see the diferent sizes on the /components page.